pkg/polylog/context.go

```diff
@@ -1,9 +1,39 @@
 package polylog
 
+import "context"
+
 // CtxKey is the key used to store the polylog.Logger in a context.Context. This
 // is **independant** of any logger-implementation-specific context key that ay
 // be used internal to any of the logger implementations. Polylog attempts to
 // provide a ubiquitous interface for storing and retrieving loggers from the
 // context but also to integrate with the underlying logger implementations as
 // seamlessly as possible.
 const CtxKey = "polylog/context"
+
+// DefaultContextLogger is the default logger implementation used when no logger
+// is associated with a context. It is assigned in the implementation package's
+// init() function to avoid potentially creating import cycles.
+// The default logger implementation is zerolog (i.e. pkg/polylog/polyzero).
+//
+// IMPORTANT: In order for the default to be populated, the polyzero package MUST
+// be part of the build. Otherwise, the polyzero package's init function will
+// neither be included in the build nor executed. If no such import exists, the
+// polyzero package can be imported for side effects only, e.g.:
+//
+// import _ "github.com/pokt-network/poktroll/pkg/polylog/polyzero"
```

Comment:
```suggestion
// 	import _ "github.com/pokt-network/poktroll/pkg/polylog/polyzero"
```

Reply:
:sunglasses: Nice - thanks!

Reply:
Hmm.. while this change improves rendering in godoc, it would appear that `goimports` does not like it. Worse yet, it will modify the line when asked to fix issues (i.e. `make go_imports`). I think it's best to take the rendering hit for now to avoid DevEx complications/confusion.